packages/wallets/src/features/cashier/modules/TransactionStatus/hooks/useRecentTransactions.ts

```diff
@@ -0,0 +1,168 @@
+import { useEffect, useMemo } from 'react';
+import moment from 'moment';
+import { useCryptoTransactions } from '@deriv/api';
+
+const useRecentTransactions = () => {
+    const { data: transactions, error, isLoading, subscribe, ...rest } = useCryptoTransactions();
+
+    useEffect(() => subscribe(), [subscribe]);
+
+    const isError = useMemo(() => !isLoading && error !== undefined, [error, isLoading]);
+
+    const recentTransactions = useMemo(
+        () =>
+            transactions?.map(transaction => {
+                const submitDate = transaction.submit_date;
+                const confirmations = transaction.confirmations;
+                const addressHash = transaction.address_hash;
+                const addressHashObscure = addressHash
+                    ? `${addressHash.substring(0, 4)}....${addressHash.substring(addressHash.length - 4)}`
+                    : 'Pending';
+                const transactionHash = transaction.transaction_hash;
+                const transactionHashObscure = transactionHash
+                    ? `${transactionHash.substring(0, 4)}....${transactionHash.substring(transactionHash.length - 4)}`
+                    : 'Pending';
+
+                const depositStatusColorMapper = {
```

Comment:
All of the mappers that are static, why dont we move it outside of react lifecycle (outside hook)